src/main.rs

```diff
@@ -1113,6 +1126,10 @@ mod tests {
             // Start the daemon
             let mut daemon_child = Command::new(vubridge_path.as_str()).spawn().unwrap();
 
+            let vhost_user_net_device = format!(
+                "mac={},sock=sock=/tmp/vubr.sock",
+                guest.network.l2_guest_mac
```

Comment:
Why are you using `l2` parameters? You're not in a use case that requires `l2`.

Reply:
oh, I make use of it to assign IP for vhost-user-net device, I changed network-config under test-data/cloud-init/ubuntu to add one interface for guest.